ext/zlib/zlib_filter.c

```diff
@@ -327,7 +327,7 @@ static php_stream_filter *php_zlib_filter_create(const char *filtername, zval *f
 				/* log-2 base of history window (9 - 15) */
 				zend_long tmp = zval_get_long(tmpzval);
 				if (tmp < -MAX_WBITS || tmp > MAX_WBITS + 32) {
-					php_error_docref(NULL, E_WARNING, "Invalid parameter give for window size. (" ZEND_LONG_FMT ")", tmp);
+					php_error_docref(NULL, E_WARNING, "Invalid parameter given for window size. (" ZEND_LONG_FMT ")", tmp);
```

Comment:
I wonder wether we should remove the period right away, i.e.
```suggestion
					php_error_docref(NULL, E_WARNING, "Invalid parameter given for window size (" ZEND_LONG_FMT ")", tmp);
```

Reply:
It does look weird, especially with the value between parenthesis after the period. I'll push another commit

Reply:
@cmb69 I applied this to messages that didn't have the issue with "give" vs "given" as well